core/domain/stats_services.py

```diff
@@ -1114,7 +1114,7 @@ def get_learner_answer_details(entity_type, state_reference):
 
     Returns:
         LearnerAnswerDetails. The learner answer domain object
```

Comment:
Probably should be `LearnerAnswerDetails|None`

Reply:
Done